aws/ci_build.rake

```diff
@@ -143,9 +143,9 @@ end
 task :code_studio_task do
   HipChat.log 'Cleaning code-studio package'
   # get rid of any symlink to a built package
-  RakeUtils.system 'rm', dashboard_dir('public/code-studio')
+  RakeUtils.system 'rm', dashboard_dir('public/code-studio') if File.exist?(dashboard_dir('public/code-studio'))
   # also delete the package itself
-  RakeUtils.system 'rm -rf', dashboard_dir('public/code-studio-package')
+  RakeUtils.system 'rm -rf', dashboard_dir('public/code-studio-package') if File.exist?(dashboard_dir('public/code-studio-package'))
```

Comment:
1. maybe use [`FileUtils`](http://ruby-doc.org/stdlib-2.2.2/libdoc/fileutils/rdoc/FileUtils.html) (`FileUtils.rm`/`FileUtils.rm_rf`) instead of shelling out via `RakeUtils` for these commands?
2. conditionals shouldn't be necessary to ignore nonexistend files, that's what the `-f` option is for. You probably just need to add `-f` to the first line here.